pdbg/switch_pdbg_backend.C

```diff
@@ -17,53 +41,76 @@ void pdbgLogCallback(int, const char* fmt, va_list ap)
     std::string logstr(logData.begin(), logData.end());
     std::cout << logstr << std::endl;
 }
-static bool flag = false;
-void myGetScom(enum pdbg_backend backend)
+
+void myGetScom(std::string_view backendString, enum pdbg_backend backend)
 {
-    // initially set the pdbg backend to sbefifo
+    // Set the pdbg backend to "backend"
     if (!pdbg_set_backend(backend, "p10"))
     {
         std::cout << "Failed to set pdbg backend to " << backend << std::endl;
     }
 
-    constexpr auto devtree = "/var/lib/phosphor-software-manager/pnor/rw/DEVTREE";
+    constexpr auto devtree =
+        "/var/lib/phosphor-software-manager/pnor/rw/DEVTREE";
     // PDBG_DTB environment variable set to CEC device tree path
-    if (setenv("PDBG_DTB", devtree, 1)) 
-    {   
+    if (setenv("PDBG_DTB", devtree, 1))
+    {
         std::cout << "Failed to set PDBG_DTB: " << strerror(errno) << std::endl;
-    }   
+    }
 
     constexpr auto PDATA_INFODB_PATH = "/usr/share/pdata/attributes_info.db";
     // PDATA_INFODB environment variable set to attributes tool  infodb path
-    if (setenv("PDATA_INFODB", PDATA_INFODB_PATH, 1)) 
-    {   
-        std::cout << "Failed to set PDATA_INFODB: ({})" << strerror(errno) << std::endl;
-    }   
-
-    //initialize the targeting system 
-    if (!pdbg_targets_init(NULL))
-    {   
+    if (setenv("PDATA_INFODB", PDATA_INFODB_PATH, 1))
+    {
+        std::cout << "Failed to set PDATA_INFODB: ({})" << strerror(errno)
+                  << std::endl;
+    }
+
+    // initialize the targeting system
+    if (!pdbg_targets_init(nullptr))
+    {
         std::cout << "pdbg_targets_init failed" << std::endl;
     }
-    if(!flag) 
+
     {
+        std::cout << "\n Probing all targets" << std::endl << std::endl;
```

Comment:
Are the cout statements added intentionally in the commits?

Reply:
> 

Yes, it is just a standalone test application so for debugging purposes console output statements are intentional